easybuild/easyconfigs/t/TomoPy/TomoPy-1.11.0-foss-2020b.eb

```diff
@@ -0,0 +1,53 @@
+# Authors: Joachim Hein, Zdenek Matej, Lund University
+
+easyblock = 'PythonPackage'
+
+name = 'TomoPy'
+version = '1.11.0'
+
+github_account = 'tomopy'
+homepage = "https://tomopy.readthedocs.io/en/latest/"
+description = """
+Tomographic reconstruction creates three-dimensional views of an object by 
+combining two-dimensional images taken from multiple directions, for example, 
+this is how a CAT (computer-aided tomography) scanner generates 3D 
+views of the heart or brain.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('scikit-image', '0.18.1'),
+    ('OpenCV', '4.5.1', '-contrib'),
+    ('imkl', '2020.4.304'),
```

Comment:
Hm, this might a a non issue, but it worries me to mix in another BLAS here.

Reply:
@Micket and @jhein32 

from what I know, [TomoPy](https://github.com/tomopy/tomopy) is heavily relaying on FT. They were using `fftw` long time ago but later they switched to `mkl_fft` as you can see also here in their conda env definition, e.g. [linux-38.yml](https://github.com/tomopy/tomopy/blob/master/envs/linux-38.yml). Note there is also `nomkl` version relying on conda `nomkl` (?).

Note also that from a discussion with leading scientific developers, it is known TomoPy has mkl optimizations included. It can even profit from the intel compiler. However we were preferring to submit the foss EasyConfigs first as having it in the foss toolchains allows to combine TomoPy with more sw.